target/scripts/build/packages.sh

```diff
@@ -43,12 +74,19 @@ function _install_postfix() {
 function _install_packages() {
   _log 'debug' 'Installing all packages now'
 
-  ANTI_VIRUS_SPAM_PACKAGES=(
-    amavisd-new clamav clamav-daemon
-    pyzor razor spamassassin
+  local ANTI_VIRUS_SPAM_PACKAGES=(
+    clamav clamav-daemon
+    # spamassassin is used only with amavisd-new and pyzor/razor
+    # are used by spamassasin
+    amavisd-new spamassassin pyzor razor
+    # the following packages are all for Fail2Ban
+    fail2ban python3-pyinotify python3-dnspython
```

Comment:
Add context on the complimentary fail2ban packages for why they're installed so this information is not lost to any future maintenance that would also try to remove them: https://github.com/docker-mailserver/docker-mailserver/pull/3403#discussion_r1306581431

```suggestion
    # the following packages are all for Fail2Ban
    # https://github.com/docker-mailserver/docker-mailserver/pull/3403#discussion_r1306581431
    fail2ban python3-pyinotify python3-dnspython
```